diff --git a/node_modules/@blueskyproject/tiled/dist/tiled.es.js b/node_modules/@blueskyproject/tiled/dist/tiled.es.js
index 0f44de0..21ca67e 100644
--- a/node_modules/@blueskyproject/tiled/dist/tiled.es.js
+++ b/node_modules/@blueskyproject/tiled/dist/tiled.es.js
@@ -3193,7 +3193,7 @@ Wt.interceptors.response.use(
       try {
         const r = Kv();
         if (r) {
-          const i = (await Wt.post(`${gr}/auth/refresh`, {
+          const i = (await Wt.post(`${gr}/auth/session/refresh`, {
             refresh_token: r.refreshToken
           })).data.access_token;
           return lO(r.refreshToken, i), _f(i), Wt(t);
@@ -3209,7 +3209,7 @@ Wt.interceptors.response.use(
 const Jv = () => {
   const e = window.location.hostname, t = window.location.protocol;
   try {
-    return `${t}//${e}:8000/api/v1`;
+    return (typeof window !== "undefined" && window.__TILED_BASE_URL__) || `${t}//${e}:8000/api/v1`;
   } catch (n) {
     return console.error("error parsing VITE_API_TILED_URL env: ", n), `${t}//${e}:8000/api/v1`;
   }
